<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiAesCipher</class>
 <widget class="QWidget" name="UiAesCipher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AES Encryption/Decryption Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>AES encrypt/decrypt</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="2" colspan="2">
       <widget class="QComboBox" name="comboBox_AesPadding">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>No padding</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ZERO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PKCS7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_AesMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>ECB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CBC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GCM (not supported)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CFB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OFB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CTR (not supported)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_IV">
        <property name="text">
         <string>AES IV</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="QTextEdit" name="textEdit_Output">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input/Output data should be 16bytes aligned, otherwise default zero padding will be applied.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_IV">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>09 18 63 1B 0D 0E 0F C6 14 DE 1D F3 33 2B C8 51</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox_AesSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>AES 128</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES 192</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AES 256</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_Key">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>2E D5 C0 15 30 78 92 7C B1 FD 43 CF C6 E6 71 CB</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="pushButton_Decrypt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Decrypt</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="pushButton_Encrypt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Encrypt</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QTextEdit" name="textEdit_Input">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;50 C7 02 53 6D 80 9F FF 25 30 3A 6E BF 90 98 8D 86 E7 F0 BF 47 F3 D5 FE 26 53 74 AB B6 C6 08 58 56 CF 01 E0 3B FE FC 14 8C 46 AE 0E 60 14 15 2E 6E B1 02 CE 0E E5 74 50 91 9E 25 8F AA D7 C4 2A 4D D7 9E 9F 5D 10 A1 73 81 86 30 10 92 27 ED 7B&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Key">
        <property name="text">
         <string>AES KEY</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_Output">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_Input">
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
